### PR TITLE
CONJ-4776 SB adds annotation to hosts

### PR DESCRIPTION
**Pull request by izgeri**
This PR updates the service broker to check its host identity (`ConjurClient.authn_login`) for an annotation of the form `platform: [platform-name]`, where `platform-name` might be `cloudfoundry`, `pivotalcloudfoundry`, `kubernetes`, or `openshift`.

If the SB host identity has a `platform` annotation, then a `[platform-name]: true` annotation will be added to all hosts added to Conjur policy by the service broker.

Only the `alt-policy-name` bind test uses the host identity with its platform annotation in the SB config, so that test was updated to confirm that the host the SB added on bind has the appropriate annotation in Conjur.

[Jenkins run](https://jenkins.conjur.net/job/cyberark--conjur-service-broker/job/CONJ-4776-SB-adds-annotation-to-hosts/)
[Jira story](https://ca-il-jira.il.cyber-ark.com:8443/browse/CONJ-4776)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+If the service broker host identity has a `platform` annotation in Conjur, hosts added to policy by the service broker will include an annotation for the platform also.
```
Comment (jtuttle) [inline CHANGELOG.md]: Ending a sentence with "also" seems grammatically iffy to me but I searched around a bit and it seems like it's perfectly acceptable so feel free to leave it!

File: ci/configure_v5.sh [hunk 2]
```diff
@@ -1,3 +1,9 @@
 #!/bin/bash -ex
 
 docker-compose exec -T conjur_5 conjurctl wait -r 30 -p 80
+
+api_key=$(docker-compose exec -T conjur_5 bash -c 'rails r "puts Role[%Q{cucumber:user:admin}].api_key" 2>/dev/null')
```
Comment (jtuttle) [inline ci/configure_v5.sh]: What prompted this change? Seems unrelated to annotations.

File: CHANGELOG.md (hunk 1 above)
Comment (izgeri) [reply CHANGELOG.md]: Updated.

File: ci/configure_v5.sh (hunk 2 above)
Comment (izgeri) [reply ci/configure_v5.sh]: Yeah, it's more for consistency - the v4 configure loads the policy, so I like the idea of v5 configure doing the same. There were some other experiments I was doing that initially prompted me to put this in here, but I think it belongs here.
